CHANGELOG.md

```diff
@@ -27,23 +30,24 @@
 
 ### Released packages
 
+- vhd-lib 4.7.0
 - @vates/multi-key-map 0.2.0
 - @vates/disposable 0.1.5
 - @xen-orchestra/fs 4.1.3
 - xen-api 2.0.0
 - @vates/nbd-client 2.0.1
 - @xen-orchestra/xapi 4.0.0
-- @xen-orchestra/backups 0.44.1
+- @xen-orchestra/backups 0.44.2
 - @xen-orchestra/backups-cli 1.0.14
 - @xen-orchestra/cr-seed-cli 1.0.0
-- @xen-orchestra/proxy 0.26.40
+- @xen-orchestra/proxy 0.26.41
 - xo-vmdk-to-vhd 2.5.7
 - @xen-orchestra/vmware-explorer 0.3.1
 - xapi-explore-sr 0.4.2
 - xo-cli 0.22.0
-- xo-server 5.128.0
+- xo-server 5.129.0
 - xo-server-netbox 1.4.0
-- xo-web 5.129.0
+- xo-web 5.130.0
```

Comment:
In the future, add newly released packages below and remove duplicates :slightly_smiling_face: